frontend/containers/share-icons/component.tsx

```diff
@@ -0,0 +1,118 @@
+import { FC, useState } from 'react';
+
+import { Facebook, Link, Mail, Twitter } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import classNames from 'classnames';
+
+import { useRouter } from 'next/router';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Toast from 'components/toast';
+
+import { ShareIconsProps } from '.';
+
+export const ShareIcons: FC<ShareIconsProps> = ({ title }) => {
+  const [copied, setCopied] = useState(false);
+  const { formatMessage } = useIntl();
+
+  const { asPath } = useRouter();
+  // Facebook don't support locales, so we use the staging url when in local
+  const shareLinkUrl = process.env.NEXT_PUBLIC_PROXY_BACKEND
+    ? `https://staging.hecoinvest.org${asPath}`
+    : `${process.env.NEXT_PUBLIC_FRONTEND_URL}${asPath}`;
```

Comment:
Is that for when you run the server locally?

If that's the case, you could probably use `process.env.NODE_ENV === 'production'` instead of `process.env.NEXT_PUBLIC_PROXY_BACKEND`. I think it's safer.

We're also hard-coding the staging URL. It may not be very important if this breaks but I wonder if that code is needed at all? If that's just for testing locally, can we remove it?